clients/fides-js/src/components/fides.css

```diff
@@ -13,6 +13,7 @@
   --fides-overlay-gpc-applied-text-color: white;
   --fides-overlay-gpc-overridden-background-color: #e53e3e;
   --fides-overlay-gpc-overridden-text-color: white;
+  --fides-overlay-brand-link-font-color: #a0aec0;
```

Comment:
Do we need the font & logo to be different colors? They are very similar here, and it'll be less error-prone to have them match.

If we want the logo color to be specified, let's have the font color reuse an existing text color. That'll make it more likely to work in existing customizations

Reply:
The colors came from the [design](https://www.figma.com/design/MFzrOaPcszfJTcx4djQqMI/HJ-157%3A-Add-Ethyca-Logo-to-CMP-%2F-Privacy-Center?node-id=0-1&node-type=canvas&t=sRzlq5SAVaCIA76p-0); the logo is in Minos which I think we want to maintain because it's a brand color.  I can change the text to match that or be in the default text color instead if we prefer, though.

Reply:
There'll be cases where the background color will be too dark for the logo to be visible, so having an option to switch it from minos -> white will need to be available is all

Reply:
Gotcha-- I've updated Minos to be a variable in `fides.css` over using an inline style so it should be easier to change and switched the text over to use the default font color instead of defining a new one:

![Screenshot 2024-11-11 at 14 35 02](https://github.com/user-attachments/assets/e1cc840c-c84d-401f-b92d-27878a71d1a8)